### PR TITLE
NAVI-[FE]/feat/Landing-Page

### PR DESCRIPTION
**Pull request by Dachikkinho**
Artist images until backend is ready!

### CODE REVIEW

File: app/components/Header/Search/Search.tsx [hunk 1]
```diff
@@ -5,17 +5,18 @@ type Props = {
...
 
-export const Search = ({ placeholder, icon, width, height, onChange }: Props) => {
+export const Search = ({ placeholder, icon, width, height, }: Props) => {
     return (
         <div className={styles.container}>
             <input
                 type="text"
                 placeholder={placeholder}
                 className={styles.mainSearch}
-                onChange={onChange}
```
Comment (lursmana1) [inline app/components/Header/Search/Search.tsx:18]: why ?

File: app/components/MainSection/MainSection.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            </div>
+                <div className={styles.wrapper}>
+                    <div>
+                        <div className={styles.carouselHeading}>
+                            <h4>Popular of the week</h4>
+                            <img src="/icons/popular.png" alt="popular-icon" width={36} height={36} />
+                        </div>
+
+                        <Carousel autoPlay infiniteLoop showArrows={false} useKeyboardArrows  showThumbs={false} className={styles.carousel} showStatus={false}>
+                            <div>
```
Comment (lursmana1) [inline app/components/MainSection/MainSection.tsx]: it has to be mapped in both occasion i think also make component or map only images

File: app/components/MainSection/TopArtist/TopArtist.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+import Link from "next/link"
+import { ArtistCard } from "./ArtistCard/ArtistCard"
+import styles from "./TopArtist.module.scss"
+
+export const TopArtist = () => {
+
+    // Placeholder until the backend is ready.
+
+
+    const artists = [
```
Comment (lursmana1) [inline app/components/MainSection/TopArtist/TopArtist.tsx]: place it in CONSTS

File: app/components/MainSection/TopArtist/TopArtist.tsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    return (
+        <div>
+            <div>
+                <div className={styles.heading}>
+                    <h5>Top Artist</h5>
+                    <img src="/icons/topartist.png" alt="topartist-icon" width={36} height={36} />
+                </div>
+            </div>
+            <div className={styles.artistsWrapper}>
+                {artists.map(artist => (
```
Comment (lursmana1) [inline app/components/MainSection/TopArtist/TopArtist.tsx]: i think Link needs key

File: app/components/SideBar/SideBar.module.scss [hunk 5]
```diff
@@ -1,8 +1,8 @@
 @import '../../Colors/Color.scss';
 
 .sidebar {
-    max-width: 300px;
-    width: 100%;
+    // no max width needed
+    width: 300px;
```
Comment (lursmana1) [inline app/components/SideBar/SideBar.module.scss]: fixed widths ?

File: app/page.module.css [hunk 6]
```diff
@@ -1,6 +1,11 @@
 .main {
   display: flex;
-  justify-content: center;
-  gap: 50px;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  min-height: 100vh;
```
Comment (lursmana1) [inline app/page.module.css]: body have 100 vh height right ? give it 100%

File: app/page.module.css [hunk 7]
```diff
@@ -1,6 +1,11 @@
...
+  max-width: 1114px;
+  /* min width here is needed */
+  min-width: 0;
```
Comment (lursmana1) [inline app/page.module.css:8]: ???

File: app/page.tsx [hunk 8]
```diff
@@ -1,9 +1,19 @@
...
+
+import { useEffect } from "react";
+import { MainSection } from "./components/MainSection/MainSection";
 import styles from "./page.module.css";
 
+
 export default function Home() {
+
+
+  useEffect(() => {
```
Comment (lursmana1) [inline app/page.tsx:11]: ?

File: app/components/MainSection/TopArtist/TopArtist.tsx [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+            </div>
+            <Link className={styles.artistsWrapper} href={"/album?album=${album.group}"}>
+                {artists.map(artist => (
+                    <ArtistCard name={artist.name} bgColor={artist.bgColor} img={artist.img} plays={artist.listens} />
+                ))}
+            </Link>
+
+            <div>
+                <div className={styles.heading}>
+                    <h5>Top Charts</h5>
```
Comment (lursmana1) [inline app/components/MainSection/TopArtist/TopArtist.tsx:48]: Styling on Headers is needed i think

File: app/components/Header/Search/Search.tsx (hunk 1 above)
Comment (Dachikkinho) [reply app/components/Header/Search/Search.tsx:18]: > why ?

Because without a search algorithm it is useless

File: app/page.module.css (hunk 7 above)
Comment (Dachikkinho) [reply app/page.module.css:8]: wrong comments from another project

File: app/page.tsx (hunk 8 above)
Comment (Dachikkinho) [reply app/page.tsx:11]: this is document.tittle

File: app/components/MainSection/TopArtist/TopArtist.tsx (hunk 9 above)
Comment (Dachikkinho) [reply app/components/MainSection/TopArtist/TopArtist.tsx:48]: div have styling. h5 attribute have default styling

**Pull request merged**